tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -143,6 +168,23 @@ test.describe.serial(`AI Lab extension installation and verification`, () => {
           .toBeTruthy();
       });
 
+      test(`Expect ${modelName} to be included in models list fetched from API`, async ({ request, page }) => {
+        await page.waitForTimeout(15_000); //Waiting for models to be ready
```

Comment:
Can we improve the wait for something specific? Like checking the catalog? No hard waiting?